tutorials/nservicebus-101/lesson-4/Solution/Sales/PlaceOrderHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+using Messages.Commands;
+using NServiceBus;
+using NServiceBus.Logging;
+using Messages.Events;
+
+namespace Sales
+{
+    public class PlaceOrderHandler :
+        IHandleMessages<PlaceOrder>
+    {
+        static ILog logger = LogManager.GetLogger<PlaceOrderHandler>();
+
+        #region UpdatedHandler
+
+        public Task Handle(PlaceOrder message, IMessageHandlerContext context)
+        {
+            logger.Info($"Received PlaceOrder, OrderId = {message.OrderId}");
+
+            // This is normally where some database logic would occur
```

Comment:
"database logic"  is a little specific. people can call a webservice, write to disk etc

how about "This is normally where some business logic would occur"